### PR TITLE
MISC: add some table ID converters

### PR DESCRIPTION
**Pull request by dannywillems**
Part of https://github.com/o1-labs/proof-systems/pull/1966/

### CODE REVIEW

File: msm/src/lookups.rs [hunk 1]
```diff
@@ -14,14 +14,14 @@ pub enum LookupTableIDs {
     /// Custom lookup table
     /// The index of the table is used as the ID, padded with the number of
     /// built-in tables.
-    Custom(usize),
+    Custom(u32),
```
Comment (querolita) [inline msm/src/lookups.rs:17]: What's the reason that motivates you to change it to u32?

File: msm/src/lookups.rs [hunk 2]
```diff
@@ -14,14 +14,14 @@ pub enum LookupTableIDs {
...
 }
 
 impl LookupTableID for LookupTableIDs {
-    fn into_field<F: Field>(self) -> F {
+    fn to_u32(&self) -> u32 {
         match self {
-            LookupTableIDs::RangeCheck16 => F::one(),
-            LookupTableIDs::Custom(id) => F::from(id as u64) + F::one(),
+            LookupTableIDs::RangeCheck16 => 1_u32,
```
Comment (querolita) [inline msm/src/lookups.rs:23]: I suppose this is equivalent to `1u32`?

**Review by querolita — APPROVED** [review]

File: msm/src/lookups.rs (hunk 2 above)
Comment (dannywillems) [reply msm/src/lookups.rs:23]: Yes

File: msm/src/lookups.rs (hunk 1 above)
Comment (dannywillems) [reply msm/src/lookups.rs:17]: To use `Fp::from(id)` :)

Comment (dannywillems) [reply msm/src/lookups.rs:17]: And I think u32 is way enough :)

**Pull request merged**